src/cli.ts

```diff
@@ -114,10 +114,12 @@ const optDefs: commandLineUsage.OptionDefinition[] = [
     defaultValue: '',
   },
   {
-    name: 'paint',
-    description: 'Include next paint in measured interval',
-    type: Boolean,
-    defaultValue: false,
+    name: 'measure',
+    description: 'Which time interval to measure. Options:\n' +
+        '- callback: bench.start() to bench.stop() (default)\n' +
```

Comment:
Consider dropping the leading hyphens. I think it's clearer without them

Reply:
IMO it's easier to read this way. Otherwise it visually blends into the next flag documentation.

Without:
```
  -s, --save string              Save benchmark JSON data to this file                                         
  --measure string               Which time interval to measure. Options:                                      
                                 callback: bench.start() to bench.stop() (default)                             
                                 fcp: first contentful paint                                                   
  --horizon string               The degrees of difference to try and resolve when auto-sampling
```

With:
```
  -s, --save string              Save benchmark JSON data to this file                                         
  --measure string               Which time interval to measure. Options:                                      
                                 - callback: bench.start() to bench.stop() (default)                           
                                 - fcp: first contentful paint                                                 
  --horizon string               The degrees of difference to try and resolve when auto-sampling 
```

Reply:
Nope, that was the first thing I tried, but command-line-usage strips leading whitespace (I even tried some other whitespace characters to trick it).

Reply:
Can we not include leading whitespace here? In that case this is good.

Reply:
Hm, the hyphens already have meaning in a flags context though. What about another delimiter, like `*`

Reply:
`*` is also used in our flags, to mean wildcards. I just removed the hyphens, not a big deal.